CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
+
+### Changed 
+* Corrected ml/stats/nodes permissions spelling from [@camAtGitHub](https://github.com/camAtGitHub)
+* Internal user usage clarification from [@reshippie](https://github.com/reshippie)
+* Remove `advanced_modules_enabled` examples from [@wjordan](https://github.com/wjordan)
+* CHANGELOG.md replaces release-notes
+
+### Security
+
+* Security patch for `org.cryptacular:cryptacular`
+* Security patch for `org.springframework:spring-core`
+* Security patch for `org.apache.kafka:kafka-clients`
+* Security patch for `org.springframework.kafka:spring-kafka-test`
+
+## [2.0.0-rc1] - 2022-04-20
```

Comment:
This section is comparable to https://github.com/opensearch-project/security/blob/main/release-notes/opensearch-security.release-notes-2.0.0.0-rc1.md#2022-04-20-version-2000-rc1 

Note; I've collapsed/omitted several updates by maintainers, it might be worthwhile discussion why we should/shouldn't include some of those items.